apps/main-cf.yml.j2

```diff
@@ -284,7 +289,7 @@ Resources:
         Rules:
           - ObjectOwnership: BucketOwnerEnforced
 
-  {% if security_environment in ('ASF', 'JPL-public') %}
+  {% if security_environment != 'JPL' %}
```

Comment:
I dislike having this condition here. I'd love to brainstorm a no-op `Principal` to use for `JPL` environments in the lower if/else blocks instead